server/controllers/addElementIntoBoard.js

```diff
@@ -1,20 +1,13 @@
 //the frontend passes the element which have to be added and the coordinates
 import checkIfValid from "../helpers/checkIfValid.js";
-import Game from "../database/gameSchema.js";
```

Comment:
It seems like a lot of the logic is being removed here, can you confirm this is intended?